### DISCUSSION TITLE
GAMA licence: discussion about evolution

### DISCUSSION THREAD

**Discussion by benoitgaudou** [gama-platform/gama#54] [General]
Up to now, GAMA licence is the GPL.
This is perfectly relevant for a software with only a desktop version, without a client-server approach.

Now that GAMA (1.9.2 or 2.0) has a fully client-server architecture, it could be the moment to rediscuss the licence.

As far as I know, GPL is perfect is perfect to make mandatory the distribution as open source of any software based on GAMA.
But with the client server architecture, it is possible to build a software using GAMA in client server mode, without sharing the source of its software. 
To prevent that (to force that the source of this new software should be open), we could consider looking at other licences.
And if I am right (and remember the discussion with OpenMole developers), the Afero GPL (agpl) licence is dedicated to the client server-based softwares.

So it could be interresting / necessary to think about the evolution of the GAMA licence.

**Comment by RoiArthurB** [comment]
I totally agree, however so far the new architecture of GAMA 2 isn't 100% a client server one. Therefore, from my knowledge, the AGPL isn't relevant if we're still releasing an IDE...

However I totally agree about starting to think about this problem with other peoples from @gama-platform/core ;)
